data/mods/Aftershock/items/gun/plasma.json

```
@@ -0,0 +1,74 @@
+[
+  {
+    "abstract": "afs_plasma_base",
+    "looks_like": "nailgun",
+    "type": "GUN",
+    "reload_noise": "pshht",
+    "name": { "str": "plasma gun" },
+    "description": "A plasma gun.",
+    "weight": "3904 g",
+    "volume": "750 ml",
+    "price": "750 USD",
+    "price_postapoc": "750 USD",
+    "to_hit": -2,
+    "bashing": 6,
+    "material": [ "plastic", "steel" ],
+    "symbol": "(",
+    "color": "yellow",
+    "ammo": [ "afs_shydrogen" ],
+    "flags": [ "NEVER_JAMS", "NON_FOULING", "NEEDS_NO_LUBE" ],
+    "ammo_effects": [ "PLASMA", "FLAME", "EMP", "EXPLOSIVE_SMALL" ],
+    "skill": "launcher",
+    "dispersion": 900,
+    "range": 6,
+    "durability": 6,
+    "loudness": 5,
+    "reload": 50,
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "emitter", 1 ],
+      [ "grip", 1 ],
+      [ "rail", 1 ],
+      [ "sights", 1 ],
+      [ "sling", 1 ],
+      [ "stock", 1 ],
+      [ "underbarrel", 1 ]
+    ]
+  },
+  {
+    "id": "afs_pam-41",
+    "type": "GUN",
+    "name": { "str": "PAM-41 2g" },
+    "copy-from": "afs_plasma_base",
+    "description": "A short and stubby 2g plasma railgun occasionally used by espatiers as a hull breaching tool.  Like larger plasma weapons, it propels a toroids of plasma that detonate on contact, but its reduced size heavily hampers its range and damage output.  Fired within an atmosphere, its very low range makes it dangerous to shooter and victim alike.",
```

```suggestion
    "description": "A short and stubby 2g plasma railgun occasionally used by espatiers as a hull breaching tool.  Like larger plasma weapons, it propels toroids of plasma that detonate on contact, but its reduced size heavily hampers its range and damage output.  Fired within an atmosphere, its very low range makes it dangerous to shooter and target alike.",
```
Not sure about "heavily hampers", although the alliteration is nice. "2g" - 2nd generation? Acceleration of 2 Gs?

---

Its 2 grams. Weight of shot hydrogen is what serves as the caliber and ammo consumpion of plasma weapons.